src/Form/TimesheetEditForm.php

```diff
@@ -19,12 +19,27 @@
 use App\Entity\Timesheet;
 use App\Form\Type\ActivityGroupedWithCustomerNameType;
 use App\Repository\ActivityRepository;
+use Symfony\Component\Translation\TranslatorInterface;
```

Comment:
can be removed

Reply:
Nop. It has to be included for the correct namespace.